### PR TITLE
Update Install Kyma doc

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- clearly state structure of the command
- restructure headlines
- ...

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/11905

### CODE REVIEW

File: docs/04-operation-guides/operations/01-install-kyma.md [hunk 1]
```diff
@@ -80,9 +81,11 @@ Optionally, you can specify from which source you want to deploy Kyma. For examp
 For example, to install Kyma from a specific version, such as `1.19.1`, run:
```
Comment (jeremyharisch) [inline docs/04-operation-guides/operations/01-install-kyma.md:81]:
```suggestion
For example, to install Kyma from a specific version, such as `2.0`, run:
```

File: docs/04-operation-guides/operations/01-install-kyma.md [hunk 2]
```diff
@@ -80,9 +81,11 @@ Optionally, you can specify from which source you want to deploy Kyma. For examp
...
 
   ```bash
-  kyma deploy --source=1.19.1
+  kyma deploy --source=2.0
   ```
 
+>**NOTE:** You can choose to write either `kyma deploy --source=2.0` or `kyma deploy --source=release-2.0`.
```
Comment (jeremyharisch) [inline docs/04-operation-guides/operations/01-install-kyma.md:87]: I would rewrite this note, in a way that it is clear for the user that `release-2.0` references a specific branch in the GitHub repository. And `2.0` references the release.
Eventho, we are sticking to the "for each release one branch" strategy, I think we should tell the user.

I.e. if a branch called `test-branch` exists in the upstream kyma repo, it is also possible to do `kyma deploy --source=test-branch`

File: docs/04-operation-guides/operations/01-install-kyma.md [hunk 3]
```diff
@@ -77,12 +78,14 @@ To install Kyma using your own domain name, you must provide the certificate and
...
 
-For example, to install Kyma from a specific version, such as `1.19.1`, run:
+- For example, to install Kyma from a specific version, such as `1.24.1`, run:
 
   ```bash
-  kyma deploy --source=1.19.1
+  kyma deploy --source=1.24.1
   ```
 
+  As Kyma follows the strategy "one branch for each release", `kyma deploy --source=2.0` and `kyma deploy --source=release-2.0` have the same result: `kyma deploy --source=2.0` deploys the specified Kyma release or patch. `kyma deploy --source=release-2.0` deploys Kyma from the branch called `release-2.0`.
```
Comment (majakurcius) [inline docs/04-operation-guides/operations/01-install-kyma.md:87]: `"one branch for each release"` - well, that depends on how you define release. We don't have separate branches for patches.
Major release: `1`, `2`, etc.
Minor release: `1.24`, `2.1`
Patch release: `1.24.6`, `2.1.3`.

Comment (jeremyharisch) [reply docs/04-operation-guides/operations/01-install-kyma.md:87]: Yes, we should be more specific here. For ever minor release we have a branch.

**Review by majakurcius — APPROVED** [review]

**Pull request merged**